collab/src/core/collab.rs

```diff
@@ -107,6 +111,45 @@ pub fn make_yrs_doc(skp_gc: bool) -> Doc {
   })
 }
 
+#[cfg(target_arch = "wasm32")]
+pub struct WebClock;
+
+#[cfg(target_arch = "wasm32")]
+impl Clock for WebClock {
+  fn now(&self) -> Timestamp {
+    let window = web_sys::window().expect("should have a window in this context");
+    let performance = window
+      .performance()
+      .expect("performance should be available");
+    performance.now() as u64
```

Comment:
This won't work exactly the same way as in yrs. See `ywasm` code: https://github.com/y-crdt/y-crdt/blob/04d82e86fec64cce0d363c2b93dd1310de05b9a1/ywasm/src/awareness.rs#L186-L190

We shouldn't use `performance.now` for global timestamps, only for precise time calculations. More explanations. See: https://stackoverflow.com/questions/30795525/performance-now-vs-date-now